.vitepress/theme/components/LoadingIcon.vue

```diff
@@ -0,0 +1,56 @@
+<script setup lang="ts">
+defineProps<{
+  className?: string;
+  hidden?: boolean;
+  size?: number;
+}>();
+</script>
+
+<template>
+  <span
+    :class="['root', className]"
+    :aria-hidden="hidden"
+    :style="{ '--size': `clamp(${size || 18}px, 8px, 100px)` }"
```

Comment:
CSS clamp syntax is `clamp(min, value, max)`, but this looks like `clamp(value, min, max)`.

Reply:
This came from the BlitzLoading icon in stackblitz repo: `${clamp(size, 8, 100)}px` so I guess it should be `clamp(8, size, 100)`?  🤔

Reply:
In `${clamp(size, 8, 100)}px`, `clamp` is a JavaScript function:
https://lodash.com/docs/4.17.15#clamp

In your code, `clamp` is the clamp CSS function:
https://developer.mozilla.org/en-US/docs/Web/CSS/clamp

Reply:
Ahh I see 😅 Okay I will fix it!!